services/frontend-service/src/ui/components/ServiceLane/ServiceLane.test.tsx

```diff
@@ -113,24 +120,38 @@ const data = [
                 createdAt: new Date(2002),
             },
         ] as Array<Release>,
```

Comment:
Is it worth to keep `as Array<Release>` in?
iirc, having it declared in line `106` should be enough.
Plus I think using `as` adds complexity and is sometimes annoying to debug. Maybe we should think about this more deeply because I think using the explicit `type` (like in line `103 to 108` a bit better.

Reply:
Yes, you're right, I fixed.
The nice things about "as ..." in tests, is that we don't need to specify so much data. But I think type safety is more important than that.